src/location/tide-phrase/index.js

```diff
@@ -1,21 +1,32 @@
 import React, { Component } from 'react'
 import glamorous from 'glamorous'
 import _ from 'lodash'
-
+import { geoCodeLocation } from '../../utils/location'
 import Styles from '../../assets/styles'
 import RemainingTideTime from './remaining-tide-time'
 
 const TidePhrase = class extends Component {
+  state = {
+    city: '',
+  }
+
+  componentDidMount() {
```

Comment:
Would I kick off this request here? I need to pass it the users location from the parent component. 

I have the request in here because the tide phrase (as of today) is the only place that cares about the city name.. 

Confused on how to get this pieced together. 

Thoughts?

Reply:
Using state here feels like overkill since it's a basic string to pass to `render`